etg/_stc.py

```diff
@@ -259,6 +260,7 @@ def run():
     #-----------------------------------------------------------------
     tools.doCommonTweaks(module)
     tools.runGenerators(module)
+    postprocess.categorise_methods()
```

Comment:
It looks like the sdist build is failing on CI/CD due to this change.  Is this really the right place where this should run?  It seems like it is depending on the docs having been built already?